src/promptflow-devkit/promptflow/_cli/_pf/_service.py

```diff
@@ -98,15 +98,23 @@ def add_parser_start_service(subparsers):
         action="store_true",
         help="If the port is used, the existing service will be terminated and restart a new service.",
     )
+    add_param_debug = lambda parser: parser.add_argument(  # noqa: E731
+        "-d",
+        "--debug",
+        action="store_true",
+        help="The debug mode flag for the promptflow service activates foreground operation, displaying debug level "
```

Comment:
make it simpler: 

Start the promptflow service in foreground, displaying debug level logs directly in the terminal